mysql-test/suite/galera/t/pxc_local_bf_abort_threadpool.test

```diff
@@ -0,0 +1,52 @@
+# ==== Purpose ===
+#
+# This test verifies that DDLs bf-abort the local transactions when threadpool
+# is enabled.
+#
+# === References ===
+#
+# PXC-3118: BF abort for MDL conflict stuck when Thread Pool used
+
+--source include/galera_cluster.inc
+
+# Check that threadpool is enabled.
+SELECT @@thread_handling;
+
+# Create auxiliary connections for the test.
+--connect(con1,localhost,root,,test, $NODE_MYPORT_1)
```

Comment:
Shouldn't this use wait_until_count_sessions?

Reply:
You're right! I will update it.

Reply:
Done